osu.Game/Skinning/SkinComponentsContainerLookup.cs

```diff
@@ -13,9 +15,14 @@ public class SkinComponentsContainerLookup : ISkinComponentLookup
         /// </summary>
         public readonly TargetArea Target;
 
-        public SkinComponentsContainerLookup(TargetArea target)
+        public readonly RulesetInfo? Ruleset;
+
+        public string GetSerialisableIdentifier() => Ruleset?.ShortName ?? "global";
```

Comment:
This is a bit weird as it's unused. But if it will be used, I'd prefer that it reuse `SkinLayoutInfo.global_identifier`.

Reply:
This was from a previous implementation, will remove for now.